xml/ha_clvm.xml

```diff
@@ -508,7 +508,7 @@ cLVM) for more information and details to integrate here - really helpful-->
    </procedure>
    <para>
     If you need further information regarding LVM2, refer to the &sls;
-    &productnumber; <citetitle>&storage;</citetitle>: <link xlink:href="dsc;/sles-12/html/SLES-all/cha-lvm.html"></link>.
+    &productnumber; <citetitle>&storage;</citetitle>: <link xlink:href="dsc;/sle-ha/12-SP5/single-html/SLE-HA-guide/#book-sleha"></link>.
```

Comment:
```suggestion
    &productnumber; <citetitle>&storage;</citetitle>: <link xlink:href="dsc;/sles-12/html/SLES-all/cha-lvm.html"></link>.
```

Reply:
Nope, this change was wrong, the LVM chapter is in a SLES guide and the old link actually is correct. This needs to point to the Storage Administration Guide for SLES 12 SP5. Using the URL snippet `sles-12` makes sure the link gets redirected to the latest SP within Code 12 which is 12-SP5. Please revert this change (or apply my suggestion above) as `https://documentation.suse.com/sles-12/html/SLES-all/cha-lvm.html` brings the reader to the correct point.